ui/src/index.test.tsx

```diff
@@ -21,6 +21,21 @@ let browser: Browser | undefined = undefined;
 
 let uiProc: ChildProcess | undefined = undefined;
 
+// Function to generate unique party names for us.
+// This should be replaced by the party management service once that is exposed
+// in the HTTP JSON API.
+let nextPartyId = 1;
+function getParty(): string {
+  const party = `P${nextPartyId}`;
+  nextPartyId++;
+  return party;
+}
+
+test('Party names are unique', async () => {
+  const parties = new Set(Array(10).fill({}).map(() => getParty()));
```

Comment:
`undefined` is usually a better choice for undefined values:
```suggestion
  const parties = new Set(Array(10).fill(undefined).map(() => getParty()));
```

Reply:
Think I liked the for loop better 😄

Reply:
@cocreature Please be my witness: @rohanjr prefers loops over functional code.

@rohanjr 😝

Reply:
https://www.youtube.com/watch?v=LJvEIjRBSDA